admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/service/persistence/module/FieldManager.java

```diff
@@ -94,6 +95,10 @@ public Object getFieldValue(Object bean, String fieldName) throws IllegalAccessE
             if (field != null) {
                 field.setAccessible(true);
                 value = field.get(value);
+                
+                if (value instanceof HibernateProxy) {
```

Comment:
No need to add this if check.  The HibernateUtils#deproxy() already does the check.